lib/mongo_client.js

```diff
@@ -84,8 +83,8 @@ const connectSchema = {
   promoteBuffers: { type: 'boolean' },
   promoteLongs: { type: 'boolean' },
   domainsEnabled: { type: 'boolean', default: false },
+  validateOptions: { type: 'boolean', default: false, deprecated: 'unknownOptionsWarningLevel' },
```

Comment:
I expected this to read:
```
validateOptions: { type: 'boolean', default: false, deprecated: true },
```
Why is there a string `unknownOptionsWarningLevel` here?

Reply:
The string was @daprahamian's suggestion, I believe, so we can emit a deprecation notice telling users that `validateOptions` is deprecated and they should use `unknownOptionsWarningLevel` instead.